src/components/TicketStatusFilter.tsx

```diff
@@ -1,27 +1,34 @@
 import { Checkbox, FormControl, InputLabel, MenuItem, Select, SelectChangeEvent, Typography } from '@mui/material';
+import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useHistory } from 'react-router-dom';
 import { TicketSearchParam } from '../api/searchApi';
 import { TicketStatus, ticketStatusValues } from '../types/publication_types/ticket.types';
 import { dataTestId } from '../utils/dataTestIds';
 
-export const TicketStatusFilter = () => {
+interface TicketStatusFilterProps {
+  isOnTasksPage: boolean;
+}
+
+export const TicketStatusFilter = ({ isOnTasksPage }: TicketStatusFilterProps) => {
   const { t } = useTranslation();
   const history = useHistory();
-  const searchParams = new URLSearchParams(history.location.search);
-  const selectedStatuses = (searchParams.get(TicketSearchParam.Status)?.split(',') ??
-    ticketStatusValues) as TicketStatus[];
 
-  const handleChange = (event: SelectChangeEvent<string[]>) => {
-    const newSelectedStatuses = event.target.value as TicketStatus[];
+  const defaultStatusFilter = (isOnTasksPage ? ['Pending'] : ticketStatusValues) as TicketStatus[];
+  const [selectedStatuses, setSelectedStatuses] = useState(defaultStatusFilter);
```

Comment:
Trengs det en egen `useState` her? Lukter litt dobbeltlagring/skyggeføring siden vi har status i URLen.
Kunne noe sånt gjort samme nytten? (tolk dette som pseudokode)
```
const statusesFromParams = searchParams.get(TicketSearchParam.Status)?.split(',')
const selectedStatuses = statusesFromParams.length > 0
  ? statusesFromParams
  : defaultStatuses
```

Reply:
Tror jeg fikk den til å virke. Måtte ta i bruk en `useEffect()` for å sette param ved første render da.